cg/cli/workflow/balsamic/store.py

```diff
@@ -22,117 +24,37 @@ def store(context):
 
 
 @store.command()
-@click.argument("config-stream", type=click.File("r"), required=False)
+@click.argument("case_id")
+@click.argument("config-stream", type=click.File("r"))
 @click.pass_context
-def analysis(context, config_stream):
+def analysis(context, case_id, config_stream):
     """Store a finished analysis in Housekeeper."""
+
     status = context.obj["db"]
-    tb_api = context.obj["tb_api"]
-    hk_api = context.obj["hk_api"]
+    case_obj = status.family(case_id)
 
-    if not config_stream:
-        LOG.error("provide a config, suggestions:")
-        for analysis_obj in tb_api.analyses(
-            status="completed", deleted=False
-        )[:25]:
-            click.echo(analysis_obj.config_path)
+    if not case_obj:
+        click.echo(click.style(f"Case {case_id} not found", fg="red"))
         context.abort()
 
-    new_analysis = _gather_files_and_bundle_in_housekeeper(
-        config_stream, context, hk_api, status, tb_api
-    )
-
-    status.add_commit(new_analysis)
-    click.echo(click.style("included files in Housekeeper", fg="green"))
-
-
-def _gather_files_and_bundle_in_housekeeper(
-    config_stream, context, hk_api, status, tb_api
-):
-    """Function to gather files and bundle in housekeeper"""
+    hk_api = context.obj["hk_api"]
 
     try:
-        bundle_data = tb_api.add_analysis(config_stream)
+        new_analysis = _gather_files_and_bundle_in_housekeeper(
+            config_stream, hk_api, status
+        )
     except AnalysisNotFinishedError as error:
         click.echo(click.style(error.message, fg="red"))
         context.abort()
-
-    try:
-        results = hk_api.add_bundle(bundle_data)
-        if results is None:
-            print(click.style("analysis version already added", fg="yellow"))
-            context.abort()
-        bundle_obj, version_obj = results
     except FileNotFoundError as error:
         click.echo(click.style(f"missing file: {error.args[0]}", fg="red"))
         context.abort()
-
-    family_obj = _add_new_analysis_to_the_status_api(bundle_obj, status)
-    _reset_action_from_running_on_family(family_obj)
-    new_analysis = _add_new_complete_analysis_record(
-        bundle_data, family_obj, status, version_obj
-    )
-    version_date = version_obj.created_at.date()
-    click.echo(f"new bundle added: {bundle_obj.name}, version {version_date}")
-    _include_files_in_housekeeper(bundle_obj, context, hk_api, version_obj)
-
-    return new_analysis
-
-
-def _include_files_in_housekeeper(bundle_obj, context, hk_api, version_obj):
-    """Function to include files in housekeeper"""
-    try:
-        hk_api.include(version_obj)
-    except hk.VersionIncludedError as error:
+    except AnalysisDuplicationError as error:
+        print(click.style("analysis version already added", fg="yellow"))
```

Comment:
Bit odd to use print over click.echo

Reply:
that is so true. I'll change it to a click echo